docs/cow-protocol/tutorials/cow-swap/native.mdx

```diff
@@ -8,7 +8,7 @@ import CancellationNotGuaranteed from '../../../partials/_cancellation_not_guara
 
 # Native tokens
 
-In blockchain networks the "native token" - for example, Ether (ETH) on Ethereum & xDAI (xDAI) on Gnosis Chain - is the primary digital currency. 
+In blockchain networks the "native token" - for example, Ether (ETH) on Ethereum and Arbitrum One & xDAI (xDAI) on Gnosis Chain - is the primary digital currency.
```

Comment:
Is it expected to use `and` for an enumeration within a group and `&` to enumerate groups like this?

Reply:
I defer this one to @mfw78

Reply:
In lieu of hearing back from @gogonimago / @marshymarsh to the contrary, I don't consider this a blocker